tests/conftest.py

```diff
@@ -16,3 +16,4 @@ def api_client():
     The test client uses the requests library for making http requests.
     """
     return TestClient(APP)
+
```

Comment:
We now require that the code be formatted with the `black` code formatter.
That's why the build is failing.
You can do `pipenv run fmt` if your branch is up to date.
or `black . -l 120` directly

Reply:
Ups, that was a mistake. I didn't want to change the `tests/conftest.py`. Sorry